### ISSUE TITLE
Weird Firefox Content Tab Button Behavior

### ISSUE THREAD

**Issue by facelessuser** [squidfunk/mkdocs-material#3977]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When using content tabs, and the overflow indicator buttons appear, Firefox will keep scrolling to the left when you scroll all the way to the right.

This occurs in the default Material Docs:

![tabbed](https://user-images.githubusercontent.com/1055125/171736497-78d5919c-0288-4cc9-a154-a7850c600cbb.gif)

### Expected behaviour

Firefox should not cause tabs to scroll back to the start when scrolled to the end.

### Actual behaviour

Firefox causes scroll to the start when scrolling to the end.

### Steps to reproduce

See GIF above

### Package versions

- Python: `3.10`
- MkDocs: NA
- Material: `8.3.0`

### Configuration

```yaml
See the above GIF. Can be replicated in current Material Docs
```

### System information

- Operating system: macOS
- Browser: Firefox

**Comment by squidfunk** [comment]
Fixed in 35210f5. I disabled scroll snapping for content tabs, as I don't want to get into fighting browser bugs here. This makes scrolling seamless, and when a tab is clicked, it is brought into view anyway, so no real need for scroll snapping.

**Comment by squidfunk** [comment]
I fixed another bug in c4cb6b4 where the scroll offset was not correctly computed when tabs are brought into view.

**Comment by squidfunk** [comment]
Released as part of 8.3.1.
